unittests/resources/xml/vm.dtd

```diff
@@ -5,7 +5,7 @@
 <!ELEMENT booleanConstraints (constraint*)>
 <!ELEMENT nonBooleanConstraints (constraint*)>
 <!ELEMENT mixedConstraints (constraint*)>
-<!ELEMENT configurationOption (name, outputString?, (prefix | preFix)?, (postfix | postFix)?, parent?, children?, impliedOptions?, excludedOptions?, defaultValue?, optional?, ((minValue, maxValue) | values)?, stepFunction?, locations?)>
+<!ELEMENT configurationOption (name, outputString?, (prefix | preFix)?, (postfix | postFix)?, parent?, children?, impliedOptions?, excludedOptions?, optional?, ((minValue, maxValue) | values)?, stepFunction?, defaultValue?, locations?)>
```

Comment:
Because the original XML format from SPL Conqueror has the defaultValue tag before the optional tag for boolean features and after the stepFunction for numeric features.
From a perspective of a numeric feature it might be better to first see which values a numeric feature can have and afterwards see the default value.
The other alternative would be to undo my change and modify the feature models so that the defaultValue tag is always before optional.